handlers/wallet_service.py

```diff
@@ -45,29 +45,29 @@ def handle_address_history(
     __: LambdaContext,
     wallet_service: Union[WalletService, None] = None,
 ) -> dict:
-
     wallet_service = wallet_service or WalletService()
 
     address: Optional[str] = event.query.get("address")
     token: Optional[str] = event.query.get("token")
+    last_tx: str = event.query.get("last_tx", None)
+    last_ts_str: str = event.query.get("last_ts", "0")
     limit_str: str = event.query.get("limit", "10")
-    offset_str: str = event.query.get("offset", "0")
 
     if address is None or token is None:
         raise ApiError("invalid_parameters")
 
     limit: int
-    offset: int
+    last_ts: int
     try:
         limit = int(limit_str)
-        offset = int(offset_str)
+        last_ts = int(last_ts_str)
     except ValueError:
         raise ApiError("invalid_parameters")
 
     if limit > 100:
```

Comment:
Shouldn't we check that it is not < 0?

Reply:
Yes we should, done in 4e60633279b2276c1cc099549bb09901e0a686b3